### PR TITLE
fix: private leak error

### PR DESCRIPTION
**Pull request by mrnossiom**

### CODE REVIEW

File: flake.nix
Comment (Hacksore) [inline flake.nix:1]: can you please remove this file?

**Review by Hacksore — CHANGES_REQUESTED** [review]

File: flake.nix
Comment (mrnossiom) [reply flake.nix:1]: Sorry, gets me every time

**Comment by mrnossiom** [comment]
Seems like CI is having trouble when bundling

**Comment by Hacksore** [comment]
I don't have access to [push to your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests) but this should fix it.
```diff
From 3d7ab87a5841ccee5db0edccc5e16e1e0f16e0bc Mon Sep 17 00:00:00 2001
From: Sean Boult <996134+Hacksore@users.noreply.github.com>
Date: Tue, 9 Jan 2024 16:52:53 -0800
Subject: [PATCH] Fix build

---
 apps/desktop/src-tauri/src/main.rs | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/apps/desktop/src-tauri/src/main.rs b/apps/desktop/src-tauri/src/main.rs
index afb82dd..4da7bd3 100644
--- a/apps/desktop/src-tauri/src/main.rs
+++ b/apps/desktop/src-tauri/src/main.rs
@@ -16,7 +16,7 @@ mod window_custom;
 use crate::commands::*;
 use constants::*;
 use std::sync::{atomic::AtomicBool, Mutex};
-use tauri::{generate_handler, Manager};
+use tauri::{generate_handler, Manager, Window, App};
 use tauri_plugin_window_state::StateFlags;
 use tray::{create_tray_items, handle_tray_events};
 
@@ -37,6 +37,7 @@ pub struct Storage {
 
 #[cfg(target_os = "macos")]
 fn apply_macos_specifics(_app: &mut App, window: &Window) {
+
   window.set_visisble_on_all_workspaces(true);
   window.set_transparent_titlebar(true, true);
 
-- 
2.43.0

```

**Comment by mrnossiom** [comment]
It's weird, the `allow edits by maintainers` option is enabled.
Let me quickly fix that

**Comment by Hacksore** [comment]
so the issue for me not being able to push is related to git lfs.
 
```
mrnossiom/master ✔ $ git push                                                                                         batch response: You need Push access to upload Git LFS objects.                                                           
error: failed to push some refs to 'https://github.com/mrnossiom/overlayed.git'
```

Related issues:
https://github.com/git-lfs/git-lfs/issues/5001
https://github.com/git-lfs/git-lfs/issues/2291

**Pull request merged**